homeworks/09-chokidar/src/importer.ts

```diff
@@ -0,0 +1,37 @@
+import fs from "fs";
+const csv = require("csvtojson");
+const { EventEmitter } = require("events");
+
+class Importer {
+  eventEmitter;
+  constructor(eventEmitter: typeof EventEmitter) {
+    this.eventEmitter = eventEmitter;
+  }
+
+  listen() {
+    this.eventEmitter.on("dirwatcher:changed", (filePath: string) =>
+      csv()
+        .fromFile(filePath)
+        .then((jsonObj: string) => {
```

Comment:
jsonObj is not a string here. Inferred type is `any[]` (`unknown[]` is also viable here)